src/main.scss

```diff
@@ -0,0 +1,11 @@
+@import "./styles/normalize.scss";
+@import "./styles/blocks/utils/_variables.scss";
+
+@import "./styles/blocks/header/header.scss";
```

Comment:
Why double nesting?
```suggestion
@import "./styles/blocks/header.scss";
```

Reply:
Because i have navigation block and main block header. I think it's more understandable